packages/AnimationsEditor-Core.package/AnimationsEditor.class/class/open.st

```diff
@@ -1,3 +1,8 @@
 as yet unclassified
 open
-	^ToolBuilder open: self new.
\ No newline at end of file
+	| var var2 |
```

Comment:
Hmm - Flag für Renaming wäre hier glaube angebracht

Reply:
added it :)